app/controllers/api/v1/add_ons_controller.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class AddOnsController < Api::BaseController
+      def create
+        service = AddOns::CreateService.new
+        result = service.create(
+          **input_params
+            .merge(organization_id: current_organization.id)
+            .to_h
+            .symbolize_keys
+        )
+
+        if result.success?
+          render(
```

Comment:
Same as https://github.com/getlago/lago-api/pull/308, we probably could add a private shared `render_add_on` method.

Reply:
Refactored :)